source/expectations/data_values/ut_compound_data_helper.pkb

```diff
@@ -541,5 +541,169 @@ create or replace package body ut_compound_data_helper is
+                              (
+                              select '||l_ut_owner ||'.ut_compound_data_helper.get_hash(ucd.item_data.getclobval()) item_hash, 
+                                      pk_hash, ucd.item_no, ucd.data_id
+                              from
+                              (
+                              select '||l_column_filter||','||l_pk_hash_sql||', item_no, data_id
+                              from  ' || l_ut_owner || q'[.ut_compound_data_tmp ucd
+                              where data_id = :self_guid or data_id = :other_guid
+                              ) ucd
+                              )ucd_out
+                       ) src
+                       on (tgt.item_no = src.item_no and tgt.data_id = src.data_id)
+                       when matched then update
+                       set tgt.item_hash = src.item_hash,
+                           tgt.pk_hash = src.pk_hash,
+                           tgt.duplicate_no = src.duplicate_no]'
+                       using a_exclude_xpath, a_include_xpath,a_join_by_xpath,a_self_data_id, a_other_data_id;    
+
+  end;
+   
+  function get_unordered(a_owner in varchar2) return varchar2 is
+    l_sql varchar2(32767);
+  begin
+    l_sql := 'with source_data as
+                       ( select t.data_id,t.item_hash,t.duplicate_no,
+                           pk_hash
+                           from  ' || a_owner || '.ut_compound_data_tmp t
+                           where data_id = :self_guid or data_id = :other_guid
+                        )           
+                       select distinct :diff_id,tmp.item_hash,tmp.pk_hash,tmp.duplicate_no
+                       from(
+                         (
+                           select t.item_hash,t. duplicate_no,t.pk_hash
+                           from  source_data t
+                           where t.data_id = :self_guid
+                           minus
+                           select t.item_hash,t. duplicate_no,t.pk_hash
+                           from  source_data t
+                           where t.data_id = :other_guid
+                         )
+                           union all
+                         (
+                           select t.item_hash,t. duplicate_no,t.pk_hash
+                           from  source_data t
+                           where t.data_id = :other_guid
+                           minus
+                           select t.item_hash,t. duplicate_no,t.pk_hash
+                           from  source_data t
+                           where t.data_id = :self_guid
+                        ))tmp';
+    return l_sql;
+  end;
+ 
+  function get_inclusion_matcher_sql(a_owner in varchar2) return varchar2 is
+    l_sql varchar2(32767);
+  begin
+    l_sql := 'with source_data as
+                       ( select t.data_id,t.item_hash,t.duplicate_no,
+                           pk_hash
+                           from  ' || a_owner || '.ut_compound_data_tmp t
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: An illegal character with code point 10 was found in this literal. ([plsql:LiteralsNonPrintableCharactersCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3ALiteralsNonPrintableCharactersCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=774&issues=AWbTzUDbO08i52vd_Xce&open=AWbTzUDbO08i52vd_Xce)

[ISSUE_KEY]: AWbTzUDbO08i52vd_Xce